src/api/spec/jobs/bs_request_action_webui_infos_job_spec.rb

```diff
@@ -45,11 +45,14 @@
 
     context 'with non existing target project' do
       let(:request) do
-        create(:bs_request_with_submit_action,
-               source_project: source_project.name,
-               source_package: source_package.name,
-               target_project: 'does-not-exist',
-               target_package: target_package.name)
+        request = build(:bs_request_with_submit_action,
+                        source_project: source_project.name,
+                        source_package: source_package.name,
+                        target_project: 'does-not-exist',
+                        target_package: target_package.name)
+        # avoid save!
```

Comment:
why? :confused: I have the comment this comment could be more explicit :unamused:

Reply:
the rspec test suite is *full* with invalid requests :(